content/mozdev/grid.rst

```diff
@@ -0,0 +1,31 @@
+public: yes
```

Comment:
Should the name of this file be `grids` instead? I forget again how these files relate to the blog posts...